src/util/lang_class.ts

```diff
@@ -74,12 +74,10 @@ export function createClass(...args) {
   }
 
   klass.superclass = parent;
-  klass.subclasses = [];
 
   if (parent) {
     Subclass.prototype = parent.prototype;
     klass.prototype = new Subclass();
-    parent.subclasses.push(klass);
```

Comment:
I am guessing this makes a lot of tests fail
Textbox etc.

Reply:
i din't see the word `subclasses` used in the codebase outside this line, so i tried to get it away. I ll look better

Reply:
I think (not sure) that callSuper uses something of that logic

Reply:
@ShaMan123 callSuper uses superclasses

Reply:
i think it was unused